src/Endpoint.js

```diff
@@ -100,7 +103,7 @@ class Endpoint {
    */
   generate() {
 
-    let fullEndpoint = Object.assign({}, JSON.parse(JSON.stringify(endpointStruct)), this.httpData);
```

Comment:
Do you know why this was added before you can remove it? I agree it should be removed.

Reply:
glad you were asking. had to check out the history. I thought it was just a mistake, but seemed to be a deep cloning mechanism. I'l fix it.